util/io/src/main/scala/sbt/IO.scala

```diff
@@ -68,18 +86,29 @@ object IO {
   }
```

Comment:
I'm prepping to clean re-install my laptop so I can't ATM, sorry.

I will if this is still like this when I'm setup again (maybe this weekend maybe in 10 days..)